outputs/http_test.go

```diff
@@ -28,22 +28,35 @@ func TestHTTPClient_Send(t *testing.T) {
 		name           string
 		method         string
 		body           string
+		headers        map[string]string
 		testServerFunc http.HandlerFunc
 		expectedError  string
 	}{
 		{
-			name:   "happy path method get",
-			method: http.MethodGet,
+			name:    "none headers in send method",
+			method:  http.MethodGet,
+			headers: nil,
```

Comment:
From what I can understand it's not a real use case. There is no option to configure template to return `nil` as result of evaluation.